### PR TITLE
SNOW-730503: pip install awscli in windows build script

### PR DESCRIPTION
**Pull request by sfc-gh-aalam**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #SNOW-730503

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

**Review by sfc-gh-sfan — COMMENTED** [review]
Why do we need to install it twice? I wonder if we could just put then in the same line of `python -m pip install --upgrade pip setuptools wheel`

File: ci/build_windows.bat [hunk 1]
```diff
@@ -22,6 +22,12 @@ if %errorlevel% neq 0 goto :error
 python -m pip install --upgrade pip setuptools wheel
 if %errorlevel% neq 0 goto :error
 
+pip install --upgrade --user awscli
+if %errorlevel% neq 0 goto :error
+
+pip install awscli
+if %errorlevel% neq 0 goto :error
+
```
Comment (sfc-gh-aalam) [inline ci/build_windows.bat:30]:
```suggestion
```

**Comment by sfc-gh-aalam** [comment]
> Why do we need to install it twice? I wonder if we could just put then in the same line of `python -m pip install --upgrade pip setuptools wheel`

Good point. Should have noticed it myself. Now removed

File: ci/build_windows.bat [hunk 2]
```diff
@@ -22,6 +22,9 @@ if %errorlevel% neq 0 goto :error
 python -m pip install --upgrade pip setuptools wheel
 if %errorlevel% neq 0 goto :error
 
+pip install --upgrade --user awscli
```
Comment (sfc-gh-sfan) [inline ci/build_windows.bat:25]: Can this be inlined with line 22? Given you also use `--user` here, maybe it's not possible.

**Review by sfc-gh-sfan — APPROVED** [review]

File: ci/build_windows.bat (hunk 2 above)
Comment (sfc-gh-aalam) [reply ci/build_windows.bat]: yep. Just did that. Hopefully it should work, otherwise we can do a `--user`

Comment (sfc-gh-aalam) [reply ci/build_windows.bat]: this works: https://jenkins.int.snowflakecomputing.com/job/BuildPyConnector-Win/

**Pull request merged**